h/browser/chrome/test/hypothesis-chrome-extension-test.js

```diff
@@ -198,44 +205,74 @@ describe('HypothesisChromeExtension', function () {
       it('injects the sidebar if the tab is active', function () {
         var tab = createTab();
         fakeTabState.isTabActive.withArgs(1).returns(true);
-        onUpdatedHandler(tab.id, {status: 'complete'}, tab);
-        assert.calledWith(fakeSidebarInjector.injectIntoTab, tab);
+        return onUpdatedHandler(tab.id, {status: 'complete'}, tab).then(
```

Comment:
You can use something like [chai-as-promised](https://github.com/domenic/chai-as-promised/) to reduce this boilerplate.